docs/collect/system-metrics.md

```diff
@@ -43,14 +43,11 @@ First, [download windows_exporter](https://github.com/prometheus-community/windo
 with the following collectors enabled, changing `0.14.0` to the version you downloaded.
 
 ```powershell
-windows_exporter-0.14.0-amd64.exe --collectors.enabled="cpu,memory,net,logical_disk,os,system,logon,tcp,process"
+windows_exporter-0.14.0-amd64.exe --collectors.enabled="cpu,memory,net,logical_disk,os,system,logon,tcp"
```

Comment:
Can we have a link instead of "First, download windows_exporter and run it with the" (including this code block) with https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#requirements?

Reply:
because we have download step there

Reply:
Done.

Reply:
My bad, I think we should use learn links 

> https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/wmi#requirements

Reply:
Link updated.